plugins/heroku/src/tasks/review.ts

```diff
@@ -28,12 +29,16 @@ options:
       }
 
       const pipeline: HerokuApiResPipeline = await herokuClient.get(`/pipelines/${this.options.pipeline}`)
+      
+      await setStageConfigVars(this.logger, 'review', 'production', pipeline.id)
```

Comment:
Because this was `continuous-integration` in error - it's currently `production` in [n-heroku-tools](https://github.com/Financial-Times/n-heroku-tools/blob/176be885a2233778929a2ec43694a7daf9c66005/tasks/configure.js#L37-L41)